bin/studio-common

```diff
@@ -28,6 +28,8 @@ DOC
   exit 1
 fi
 
+# Install curl silently (we can't use `install_if_missing` because it is not yet loaded)
+hab pkg install -b core/curl >/dev/null
```

Comment:
Should we add `core/curl` to the plan rather than installing it everytime it is sourced?

Reply:
This is specifically for the time we enter the studio. The plan has nothing to do at this point yet.

Reply:
I mean the plan for ci-studio-common. So when you install the ci-studio-common package it comes with curl. Then we just reference that copy of curl?

Reply:
uh, interesting. I like that. Let me try it!